gulpfile.js

```diff
@@ -376,6 +378,50 @@ export function restartServer(done) {
   }
 }
 
+export function runGithubChangelogGenerator() {
+  return execa.command('github_changelog_generator -u educandu -p educandu --since-tag 0.1.0', { stdio: 'inherit' });
+}
+
+export async function generateJiraLinks() {
+  let fileStream = null;
+  const newLines = [];
+  try {
+    fileStream = createReadStream('CHANGELOG.md');
+    const lineReader = readline.createInterface({
+      input: fileStream,
+      crlfDelay: Infinity
+    });
+
+    for await (const line of lineReader) {
+      let jiraLine = null;
+      if ((/^- EDU-\d+/).test(line)) {
+        const jiraNumber = line.match(/EDU-\d+/);
+        jiraLine = line.replace(/^- EDU-\d+/, `- [${jiraNumber}](https://educandu.atlassian.net/browse/${jiraNumber})`);
```

Comment:
If you want to replace all occurrences of JIRA numbers within one line (e.g. in smth. like "- EDU-100: Revert changes from EDU-99", you could use the callback of `String.prototype.replace`:

~~~js
jiraLine = line.replace(/EDU-\d+/g, num => `[${num}](https://educandu.atlassian.net/browse/${num})`);
~~~

You could also just run this for each line (no need for the `if` and the `line.match`), thus simplifying the code.

Reply:
Ok